corebehrt/main_causal/helper/build_tree.py

```
@@ -0,0 +1,141 @@
+import csv
+import pickle
```

Ruff (0.8.2)

`pickle` imported but unused

Remove unused import: `pickle`

(F401)